### DISCUSSION TITLE
Gray cod

### DISCUSSION THREAD

**Discussion by harDkawaii** [hneemann/Digital#1252] [Q&A]
hello! i m a student and i have some trouble at grady cod 
![image](https://github.com/hneemann/Digital/assets/158105157/d3ec604f-6aae-4ade-b711-4eb0635ca449)

how can i run this cod at digital? thanks for any advice

**Comment by hneemann** [comment]
You can find a FSM editor at Analyse->Finite State Machine which allows you to draw the given counter, then create a transition table and from that the circuit. It would look like that: :smile: 

![sm](https://github.com/hneemann/Digital/assets/7190563/a4ab10dd-88e7-4381-92a5-df4d072f6c52)
